### ISSUE TITLE
Building the omni-completion list from the terminal

### ISSUE THREAD

**Issue by tvatter** [jalvesaq/Nvim-R#402]
In #278, you mentioned that:

> Before starting R for you, when Nvim-R detects that nvimcom either is not installed yet or is outdated, it runs R three times:
>
> to build the nvimcom package;
> to install the nvimcom package;
> to build the omni completion lists and syntax scripts.

Is there any way to perform the third step in the terminal before even starting nvim? Or at least in nvim without starting R?

The reason that I ask is because, if the omni completion list isn't already created and [ncm-R](https://github.com/gaalcaras/ncm-R) is installed, R does not start properly.

Note that I setup everything using an install script that calls `nvim +PlugInstall`, and I currently need to 

* comment the line `Plug 'gaalcaras/ncm-R'`
* open an R file with nvim
* start R to build the omni completion list
* close nvim
* uncomment the line `Plug 'gaalcaras/ncm-R'`

Hence, ideally, I would build the omni completion list before I even start nvim the first time...

**Comment by jalvesaq** [comment]
I can't replicate the issue. I uninstalled nvimcom and deleted the directory `~/.cache/Nvim-R`. Ncm-R raises the exception (I have to type `:messages` to see the complete error message):
```
[ncm_r@yarp] FileNotFoundError: Could not find any omnils_* files in .cache/Nvim-R.                                                                              
```
But, then, I can start R normally.

**Comment by tvatter** [comment]
Hmmm, let me try again, because I was able to replicate this on two different systems, and @tnagler had the same problem too.

**Comment by tvatter** [comment]
Strange, did the same as you (i.e., deleting `~/.cache/Nvim-R` and uninstalling nvimcom), and all I got was

`[ncm_r@yarp] Job is dead. cmd=['/usr/bin/python3', '-u', '/home/tvatter/.local/share/nvim/plugged/nvim-yarp/pythonx/yarp.py', '/tmp/nvim7aXqsR/0', 7, 'ncm_r']`

I was then able to start R normally too, so it's not a big deal. I'll watch this carefully next time I reinstall, as I was not able to reproduce the bug that I noticed (twice actually, and @tnagler got the same issue).
